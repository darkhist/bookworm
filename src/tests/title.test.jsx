import React from 'react';
import { shallow } from 'enzyme';

import Title from '../components/title/title';

describe('Title', () => {
  it('should render without crashing', () => {
    const title = shallow(<Title />);
    expect(title).toMatchSnapshot();
  });

  it('should have the correct title', () => {
    const title = shallow(<Title />);
    expect(title.text()).toContain('Bookworm');
  });
});
