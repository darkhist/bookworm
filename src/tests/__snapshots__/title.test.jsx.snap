// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should render without crashing 1`] = `
<header>
  <styled.h1>
    <span
      aria-label="book emoji"
      role="img"
    >
       📚 
    </span>
    Bookworm
    <span
      aria-label="bug emoji"
      role="img"
    >
       🐛 
    </span>
  </styled.h1>
</header>
`;
